spec/features/likes_feature_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+feature 'liking photos' do
+
+  before do
+    photo = Photo.create(title: 'mountains', caption: 'we be caption')
+    photo.comments.create(comment: 'great view!')
+  end
+
+  scenario 'a user can like a photo' do
+    visit '/'
+    click_link 'Like mountains'
+    expect(page).to have_content('1 like')
+  end
+end
+
+def sign_up
```

Comment:
Check this out for writing better help methods of this sort: https://gist.github.com/ptolemybarnes/2dfda607b85d01e113b0